mgc/discrim/discrimTwoSample.py

```diff
@@ -0,0 +1,170 @@
+from sklearn.metrics import euclidean_distances
+from ._utils import _CheckInputs
+import numpy as np
+import random
+from .base import DiscriminabilityTest
+from scipy._lib._util import MapWrapper
+
+
+class DiscrimTwoSample(DiscriminabilityTest):
+    r"""
+     A class that performs a two-sample test for whether the discriminability is different for that of
+     one dataset vs another, as described in [#1Dscr]_.
+    
+    Parameters
+    ---------- 
+    is_dist : Boolean, optional (default: False)
+        whether `x1` and `x2` are distance matrices or not.
+    remove_isolates : Boolean, optional (default: True)
+        whether remove the samples with single instance or not.
+
+    See Also
+    --------
+    DiscrimOneSample : Onesample test for accessing discriminability of a single measurement
```

Comment:
One sample not Onesample

Reply:
https://github.com/neurodata/mgc/blob/8950dbc95e34818b226e177a9331fa37b7fbbaaf/mgc/discrim/discrim_two_samp.py#L23-L25